src/pybind/mgr/dashboard/services/nvmeof_client.py

```diff
@@ -58,11 +58,12 @@ def __init__(self, gw_group: Optional[str] = None, traddr: Optional[str] = None)
             if root_ca_cert:
                 client_key = NvmeofGatewaysConfig.get_client_key(service_name)
                 client_cert = NvmeofGatewaysConfig.get_client_cert(service_name)
+                server_cert = NvmeofGatewaysConfig.get_server_cert(service_name)
```

Comment:
Shall we do this outside of the if block? Basically get the server_cert and then check if its not empty and then get the client key and cert? root_cert at this point is a dummy thing which is not being used at all.. so there's no point in using it. We can adapt later once grpc api starts accepting root_cert